testmodule/miner.nim

```diff
@@ -0,0 +1,8 @@
+import chronos
+import pkg/ethers/providers/jsonrpc
+
+
+proc mineBlocks*(provider: JsonRpcProvider, blks: int) {.async.} =
+  for i in 1..blks:
+    discard await provider.send("evm_mine")
```

Comment:
You can add a [parameter to evm_mine](https://trufflesuite.github.io/ganache/#evm_mine) with the number of blocks that you'd like to mine.

Reply:
The documentation is odd, as the argument name is `timestamp` yet the example shows that we can specify a number of blocks to mine. I tried a few flavours of this, but none seem to work:
```nim
discard await provider.send("evm_mine", @[%*{"blocks":blks}])
discard await provider.send("evm_mine", @[%*[{"blocks":blks}]])
```
The error states it is expecting a number (like a `timestamp` would indicate):
```
Errors encountered in param 0: Invalid value [{"blocks":3}] supplied to : (QUANTITY | number)/0: QUANTITY, Invalid value [{"blocks":3}] supplied to : (QUANTITY | number)/1: number
```

So then I also tried just passing in blocks by itself:
```nim
discard await provider.send("evm_mine", @[%*blks])
```
which errors with
```
Timestamp 3 is lower than previous block's timestamp 1652879359
```
I guess I'll just keep the loop for now and remove the sleep.